### PR TITLE
livingDoc implemented for test reports

### PR DESCRIPTION
**Pull request by svaleria**

### CODE REVIEW

File: .github/workflows/Main.yml [hunk 1]
```diff
@@ -1,28 +1,47 @@
...
+        echo "TODO-LY-EMAIL=${{secrets.TODO_LY_EMAIL}}" >> $GITHUB_ENV
+        echo "TODO-LY-PASS=${{secrets.TODO_LY_PASS}}" >> $GITHUB_ENV
+        echo "DriverMode=Headless" >> $GITHUB_ENV
+    - name: Run UI tests
+      run: dotnet test 3.Tests/UI --no-build --no-restore --filter "Category=Smoke | Category=Regression" --logger "trx;LogFileName=${{ github.run_number }}.trx"
+
+    - name: Create Livingdoc reports
+      if: always()
+      run: livingdoc feature-folder 3.Tests/UI --output 3.Tests/UI/ReportOutputs/LivingDoc.html
+    - name: Upload LivingDoc report file
```
Comment (sebasonti-jalauni) [inline .github/workflows/Main.yml:35]: I guess this PR depends on PR #51, here you should also make sure you are including the screenshots in the artifact

File: .github/workflows/Main.yml [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
+      uses: actions/upload-artifact@v3
+      with:
+        name: Run#${{ github.run_number }}_LivingDoc.html
+        path: 3.Tests/UI/ReportOutputs/LivingDoc.html
+    - name: Upload .trx report file
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: Run#${{ github.run_number }}_report.trx
+        path: 3.Tests/UI/TestResults/${{ github.run_number }}.trx
```
Comment (sebasonti-jalauni) [inline .github/workflows/Main.yml:46]: Why do we need to upload the .trx file as an artifact?

Comment (svaleria) [reply .github/workflows/Main.yml]: deleted trx file

File: .github/workflows/Main.yml (hunk 1 above)
Comment (svaleria) [reply .github/workflows/Main.yml:47]: solved

**Review by gioacchinogioffre — APPROVED** [review]

**Review by Luisjor — APPROVED** [review]

**Pull request merged**